libbeat/paths/paths.go

```diff
@@ -38,27 +40,43 @@
 package paths
 
 import (
+	"flag"
 	"fmt"
 	"os"
 	"path/filepath"
 )
 
+var (
+	// TODO: remove this flag in 8.0 since it should be replaced by system.hostfs configuration option (config.HostFS)
+	// HostFS is an alternate mountpoint for the filesytem root, for when metricbeat is running inside a container.
```

Comment:
Consider adding also a deprecation warning when this flag is used.

Reply:
I could have sworn we had a special log API for that, but I might be wrong?

Reply:
`cfwarn.Deprecate()`?